packages/apollo-link-state/src/__tests__/client.ts

```diff
@@ -17,7 +17,7 @@ describe('non cache usage', () => {
     `;
 
     const link = new ApolloLink(() => Observable.of({ data: { field: 1 } }));
-    const local = withClientState({});
+    const local = withClientState({ resolvers: {} });
```

Comment:
This should just be `withClientState()` probably, right? In that case all it does it strip out fields with `@client` directives, which still seems useful.

Reply:
Yeah that's a good point, I can make that change!